src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -34,8 +34,11 @@
   "minecolonies.config.generatesupplyloot": "Supply loot",
   "minecolonies.config.generatesupplyloot.comment": "Whether to generate supply ship/camps in chests occasionally.",
 
-  "minecolonies.config.forceloadcolony": "Force Load Colony",
-  "minecolonies.config.forceloadcolony.comment": "If part of the colony is loaded by an owner/officer, the entire colony should be kept loaded.",
+  "minecolonies.config.forceloadcolony": "Chunk Load Colony",
+  "minecolonies.config.forceloadcolony.comment": "If part of the colony is loaded by an owner/officer, the colony should be kept loaded.",
```

Comment:
```suggestion
  "minecolonies.config.forceloadcolony.comment": "If part of the colony is loaded by an owner/officer, should other parts of the colony be kept loaded? (Set how many chunks are loaded with the "Colony Chunk Loading Strictness" option.),
```

Reply:
thats actually not exactly what is meant, the way it works is that when a player loads the chunks and someone with permissions to keep the colony loaded that those chunks then will not unload. So its not the "other parts"

Reply:
how abt this?